### PR TITLE
Push latest version of `eslint-plugin-ember`

### PR DESCRIPTION
**Pull request by FadhlanR**
This PR is aimed to use the last version of`eslint-plugin-ember` to fix this [issue](https://github.com/ember-cli/eslint-plugin-ember/issues/2146).

### CODE REVIEW

File: packages/host/package.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
     "lint:fix": "concurrently \"pnpm:lint:*:fix\" --names \"fix:\"",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
+    "lint:js": "eslint . --quiet --cache",
```
Comment (FadhlanR) [inline packages/host/package.json]: There was a deprecation warning that we can ignore but I didn't find a way to turn it off from `eslintrc.js` so I chose to use `--quiet` options to turn off `warning` message.

**Review by jurgenwerk — APPROVED** [review]

File: packages/boxel-motion/test-app/app/controllers/split-view.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ function addPanelAnimation(
         width: `${fixedContentWidth}px`,
       },
       timing: {
-        behavior: new StaticBehavior({ fill: true }),
```
Comment (backspace) [inline packages/boxel-motion/test-app/app/controllers/split-view.ts]: Is this change needed? It seems like a valid flag 🤔

**Review by backspace — APPROVED** [review]
Thanks for reporting the bug 🎉
